src/cobra/core/model.py

```diff
@@ -371,10 +375,7 @@ def copy(self):
                 new_met = new.metabolites.get_by_id(metabolite.id)
                 new_reaction._metabolites[new_met] = stoic
                 new_met._reaction.add(new_reaction)
-            for gene in reaction._genes:
-                new_gene = new.genes.get_by_id(gene.id)
-                new_reaction._genes.add(new_gene)
-                new_gene._reaction.add(new_reaction)
+            new_reaction.gpr = reaction.gpr.copy()
```

Comment:
Not necessary, already fixed in the previous PR. See the tests that pass already.